lemur/tests/factories.py

```diff
@@ -140,6 +140,7 @@ class CACertificateFactory(CertificateFactory):
 class InvalidCertificateFactory(CertificateFactory):
     body = INVALID_CERT_STR
     private_key = ''
+    chain = ''
```

Comment:
Needed to override this, otherwise it would cause `AssertionError` in `Certificate.check_integrity`